datalad/distribution/tests/test_publish.py

```diff
@@ -315,13 +367,21 @@ def test_publish_depends(
     # only the source has the probe
     ok_file_has_content(opj(src_path, 'probe1'), 'probe1')
     for p in (target1_path, target2_path, target3_path):
-        assert_false(exists(opj(p, 'probe1')))
+        assert_false(lexists(opj(p, 'probe1')))
     # publish to a standalone remote
     source.publish(to='target3')
-    # no others are affected
+    ok_(lexists(opj(target3_path, 'probe1')))
+    # but it has no data copied
+    ok_(not exists(opj(target3_path, 'probe1')))
```

Comment:
We need a `file_has_content` here. `exists` is true without content on "crippled filesystems".